packages/expo-in-app-purchases/ios/EXInAppPurchases/EXInAppPurchasesModule.m

```diff
@@ -422,12 +424,14 @@ - (int)errorCodeNativeToJS:(SKErrorCode)errorCode
     case SKErrorPaymentInvalid:
     case SKErrorPaymentNotAllowed:
     case SKErrorPaymentCancelled:
+    case SKErrorOverlayCancelled:
       return 1;
     case SKErrorStoreProductNotAvailable:
       return 6;
     case SKErrorCloudServiceRevoked:
     case SKErrorCloudServicePermissionDenied:
     case SKErrorCloudServiceNetworkConnectionFailed:
+    case SKErrorOverlayTimeout:
```

Comment:
Instead of "CLOUD_SERVICE" as the error, it seems like "SERVICE_TIMEOUT" (4) would be a closer match.